test-apps/display-test-app/android/imodeljs-test-app/app/src/main/java/com/bentley/imodeljs_test_app/MainActivity.kt

```diff
@@ -192,9 +185,9 @@ class MainActivity : AppCompatActivity() {
                 }
             }, "DTA_Android")
 
-        host.setFrontend(frontend)
-        setContentView(frontend)
-        frontend.loadEntryPoint()
+        host.webView = webView
+        setContentView(webView)
+        host.loadEntryPoint(env.optStringNotEmpty("IMJS_DEBUG_URL") ?: "https://${WebViewAssetLoader.DEFAULT_DOMAIN}/assets/frontend/index.html", args)
```

Comment:
I think it would be cleaner and less confusing if the creation and initialization of the args variable was moved down to be right above this line. As it is, args gets created an initialized in the spot it was used before. Since it is no longer used before here, I think that should move here.

Reply:
Fixed